### ISSUE TITLE
Plugin configuration is not prefixed when started standalone

### ISSUE THREAD

**Issue by lucidNTR** [eHealthAfrica/kazana#137]
@jo ( please add patricias successor ;-) )

when i add a config option to a kazana plugin into index.js like this:
```
module.exports = {
 name: 'kazana-email-box',

 config: {
   emailAccount: '"me@gmail.com":password@imap.gmail.com:993',
   cacheDirectory: './imap_cache',
   days: 7,
   timeout: 3000000,
   pollingBreak: 100000,
   filenameFilter: '^[^\.](.+\.xls.?)|(.+\.zip)$'
 },
```

​[7:54] 
and i start the parent app, then it gets atuomatically prefixed by the plugin namespace eg. kazana-email-box

​[7:55] 
but if i run the plugin standalone eg. for testing, the paramenters are added to the root of the server configuration without a namespace

​[7:55] 
is this a bug?

​[7:55] 
it makes it hard to test modules and consistently access default configuration

**Comment by gr2m** [comment]
This is as designed, when you start a plugin standalone, it behaves like an app, and it serves at `/` root path by default. 

I agree that makes things complicated if you want to run the plugins in separate processes locally. If the goal is the have 1:1 the same setup as on production, wouldn’t it be better, to use the same docker image somehow?

**Comment by jo** [comment]
Do you both talk about the same prefix?
This sounds to me like @gr2m talks about prefixed *routes* (`/kazana-email-box/my-route`) while @lucidNTR means *config* space prefixing, eg. the example above becomes `config['kazana-email-box'].emailAccount`.

**Comment by lucidNTR** [comment]
exactly

**Comment by gr2m** [comment]
ahhhh okay, same as described here, right? https://github.com/eHealthAfrica/kazana-config/issues/17

**Comment by gr2m** [comment]
this is resolved with [kazana@8.5.0](https://github.com/eHealthAfrica/kazana/releases/tag/v8.5.0)
